xml/art_solution_automation.xml

```diff
@@ -0,0 +1,438 @@
+    </para>
+    <itemizedlist>
+      <title>Scenarios for <package>saphanabootstrap-formula</package></title>
+      <listitem>
+        <para>
+          &hana; single instance automated deployment.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Two &hana; instances on system replication setup.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Two &hana; instances on system replication and cluster scale-up
+          performance-optimized configuration.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Two &hana; instances on system replication setup, cluster scale-up
+          cost-optimized configuration, and extra &hana; installation on a secondary host.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Cloud Service Provides (Azure, AWS, and GCP) automated adjustments for each provider 
+          as best-practice.
+        </para>
+      </listitem>
+    </itemizedlist>
+    <example>
+      <title>Installation</title>
+      <screen>zypper addrepo --refresh obs://network:ha-clustering:Factory ha-clustering
+zypper install --repo ha-clustering saphanabootstrap-formula</screen>
+    </example>
+    <para>
+      The following links provides best practices:
+    </para>
+    <itemizedlist>
+      <title>Best Practices</title>
+      <listitem>
+        <para>
+          <link xlink:href="https://documentation.suse.com/sbp/all/html/SLES4SAP-hana-sr-guide-PerfOpt-12/index.html"
+            >SAP HANA System Replication Scale-Up - Performance Optimized Scenario (Code 12)</link>
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <link xlink:href="https://www.suse.com/media/white-paper/sap_hana_sr_cost_optimized_scenario_12_sp1.pdf"
+            >Setting up a SAP HANA SR Cost Optimized Infrastructure (SLES-SAP 12 SP1)</link>
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <link xlink:href="https://documentation.suse.com/sbp/all/html/SLES4SAP-hana-sr-guide-PerfOpt-15/index.html"
+            >SAP HANA System Replication Scale-Up - Performance Optimized Scenario (Code 15)</link>
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <link xlink:href="https://documentation.suse.com/sbp/all/html/SLES4SAP-hana-sr-guide-PerfOpt-12_AWS/index.html"
+            >SAP HANA High Availability Cluster for the AWS Cloud </link>
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <link xlink:href="https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/sap-hana-high-availability"
+            >High availability of SAP HANA on Azure VMs on SUSE Linux Enterprise Server</link>
+        </para>
+      </listitem>
+    </itemizedlist>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-autom-hana-and-netweaver-deploy">
+    <title>&hana; and Netweaver Automated Configuration</title>
```

Comment:
I would remove the `HANA` part from this header if we are going to talk about the `sapnwbootstrap-formula`. They are quire independent

Reply:
Ok, done. :+1: